tck/src/main/java/org/eclipse/microprofile/reactive/messaging/tck/ProcessorChainTest.java

```diff
@@ -39,7 +39,7 @@
     @Deployment
     public static Archive<JavaArchive> deployment() {
         JavaArchive archive = ShrinkWrap.create(JavaArchive.class)
-            .addClasses(BeanWithChain.class)
+            .addClasses(BeanWithChain.class, ArchiveExtender.class)
```

Comment:
Not sure I understand why `ArchiveExtender` needs to be inside the deployment when it's used when creating the deployment, which happens outside the container

Reply:
Yeah, I was a bit confused about that too.

The class can be loaded ok without `ArchiveExtender` being present but during application startup, open liberty was calling `Class.getDeclaredMethods()` on the test class as part of its injection processing. This reflection call threw a `NoClassDefFoundError` for `ArchiveExtender`.

Reply:
Moving to the deployment will ensure the tck run both by standalone TCK runner (different jvm from the runtime jvm) and the embedded TCK runner (same jvm as the one running the runtime). This issue only occurs when running in the standalone TCK runner mode.